utils/update.php

```diff
@@ -99,12 +101,7 @@
         fail('pyosmium-get-changes not found or not usable');
     }
     if (!$aResult['no-update-functions']) {
-        $sSetup ='@PHP_BIN@ '. CONST_InstallPath.'/utils/setup.php';
-        $iRet = -1;
-        passthru($sSetup.' --create-functions --enable-diff-updates', $iRet);
-        if ($iRet != 0) {
-            fail('Error running setup script');
-        }
+        createFunctions($aCMDResult);
```

Comment:
This should be (new SetupFunctions(array('enable-diff-updates' => true, ...)))->createFunctions(); but I'm sure that's the part you have not gotten around yet.